src/main/build/version.h

```diff
@@ -16,7 +16,7 @@
  */
 
 #define FC_VERSION_MAJOR            2  // increment when a major release is made (big new feature, etc)
-#define FC_VERSION_MINOR            6  // increment when a minor release is made (small new feature, change etc)
+#define FC_VERSION_MINOR            5  // increment when a minor release is made (small new feature, change etc)
```

Comment:
Minor nit

Reply:
Oops, didn't mean to commit that lol. Just did that so I could use the configurator.

Reply:
Fixed and squashed